integration_test/version_test.go

```diff
@@ -27,7 +27,19 @@ func TestKrewVersion(t *testing.T) {
 	test := NewTest(t)
 
 	stdOut := string(test.Krew("version").RunOrFailOutput())
+	checkRequiredSubstrings(test, "https://github.com/kubernetes-sigs/krew-index.git", stdOut)
+}
+
+func TestKrewVersion_CustomDefaultIndexURI(t *testing.T) {
```

Comment:
I think this test is the best to keep, because it is simple and does not do any extra installation. :+1: